reconcile/cli.py

```diff
@@ -1649,6 +1649,33 @@ def terraform_cloudflare_resources(
     )
 
 
+@integration.command(short_help="Manage Cloudflare DNS using Terraform.")
+@print_to_file
+@enable_deletion(default=False)
+@threaded(default=20)
+@binary(["terraform"])
+@binary_version("terraform", ["version"], TERRAFORM_VERSION_REGEX, TERRAFORM_VERSION)
+@account_name
```

Comment:
To select a subset of resources to run against, should we support account? zone? both?

Reply:
Both :)